test/integration.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const assert = require('assert');
+const blc = require('broken-link-checker');
+const test = require('selenium-webdriver/testing');
+const webdriver = require('selenium-webdriver');
+
+const timeOut = 35000;
+
+const baseURL = process.env.VS_TEST_URL || 'https://viewsourceconf-stage.us-west.moz.works';
+console.log(`Running integration tests against ${baseURL}`)
+
+describe('The site...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+
+    it('should not have any broken links', function(done) {
+        const options = {
+            // https://github.com/stevenvachon/broken-link-checker
+            excludedKeywords: ['/2015', 'livereload'],
+            filterLevel: '3',
+        };
+
+        const errors = {};
+
+        const siteChecker = new blc.SiteChecker(options, {
+            link(result) {
+                if (result.broken) {
```

Comment:
Would the test run fail if an external site link returns a broken response? This might be OK as-is if you want to catch these things (which can often be temporary), but you could alternatively restrict it to internal links only.